go/cmd/dolt/commands/pull.go

```diff
@@ -171,11 +177,15 @@ func (cmd PullCmd) Exec(ctx context.Context, commandStr string, args []string, d
 				})
 			}
 		} else {
+			// dolt_pull procedure returns the fast-forward status as the column index 0. Not sure if there is an appropriate
+			// place to define this magic number.
```

Comment:
Good idea to define a constant. Putting it in the stored procedure seems like a good place, since that's the official source for what result set is returned. Pretty minor though, so no biggie if you don't feel like adding it and just wanna get the fix released.

Reply:
fixed.